src/org/usfirst/frc/team694/robot/Robot.java

```diff
@@ -100,8 +105,18 @@ public void disabledPeriodic() {
 
     @Override
     public void autonomousInit() {
-        Robot.drivetrain.resetEncoders(); // TEST
```

Comment:
I don't know if this should've been removed

Reply:
It says test though. I'm not sure. I'm pretty sure it should happen but...

Reply:
I think the encoders reset in robotInit of Robot.java